tests/genomic_file/test_genomic_file_models.py

```diff
@@ -206,67 +167,51 @@ def test_update_acl_only(self):
             'metadata': {}
         }
         self.indexd.Session().put.assert_any_call(
-                '{}?rev={}'.format(did, gf.rev), json=expected)
+            '{}?rev={}'.format(did, gf.rev), json=expected)
 
     def test_delete(self):
         """
         Test delete existing genomic file
         """
         # Create and save genomic files and dependent entities
-        biospecimen_id, kwargs_dict = self._create_save_genomic_files()
+        kwargs_dict = self._create_save_genomic_files()
 
         # Get genomic files for biospecimen
-        biospecimen = Biospecimen.query.filter_by(
-            kf_id=biospecimen_id).one()
-
+        biospecimen = Biospecimen.query.first()
         # Delete genomic files
         for gf in biospecimen.genomic_files:
             db.session.delete(gf)
         db.session.commit()
 
         # Check database
-        biospecimen = Biospecimen.query.filter_by(
-            kf_id=biospecimen_id).one()
+
+        biospecimen = Biospecimen.query.first()
         self.assertEqual(len(biospecimen.genomic_files), 0)
 
-    def test_delete_via_biospecimen(self):
+    def test_cascade_delete_via_biospecimen(self):
```

Comment:
Almost there, I think you need one more test like this one. Test that deleting a genomic file deletes the associated rows in BiospecimenGenomicFile.